src/index.ts

```diff
@@ -806,6 +806,11 @@ export class ValidationService {
                     submitterInput.value = submitter.getAttribute('value');
                     form.appendChild(submitterInput)
                 }
+
+                const formAction = submitter.getAttribute('formaction');
+                if (formAction) {
+                    form.action = formAction;
+                }
             }
 
             form.submit();
```

Comment:
Should we reset `form.action` to its prior value after `form.submit()`, for buttons without `formaction`?

Reply:
Ah, good idea. We should probably always reset it right after submitting it.